cmd/kubeadm/app/cmd/upgrade/plan_test.go

```diff
@@ -493,20 +513,22 @@ _____________________________________________________________________
 					},
 				},
 			},
-			expectedBytes: []byte(`Components that must be upgraded manually after you have upgraded the control plane with 'kubeadm upgrade apply':
+			expectedBytes: []byte(`
+Components that must be upgraded manually after you have upgraded the control plane with 'kubeadm upgrade apply':
+
 COMPONENT   CURRENT       AVAILABLE
-Kubelet     1 x v1.10.2   v1.11.0
+kubelet     1 x v1.10.2   v1.11.0
 
 Upgrade to the latest kubedns to coredns:
 
-COMPONENT            CURRENT   AVAILABLE
-API Server           v1.10.2   v1.11.0
-Controller Manager   v1.10.2   v1.11.0
-Scheduler            v1.10.2   v1.11.0
-Kube Proxy           v1.10.2   v1.11.0
-CoreDNS                        1.0.6
-Kube DNS             1.14.7    
-Etcd                 3.1.11    3.2.18
+COMPONENT                 CURRENT   AVAILABLE
+kube-apiserver            v1.10.2   v1.11.0
+kube-controller-manager   v1.10.2   v1.11.0
+kube-scheduler            v1.10.2   v1.11.0
+kube-proxy                v1.10.2   v1.11.0
+coredns                             1.0.6
```

Comment:
only requested to change to CoreDNS and kube-dns as these are official names.
the lowercase change for kube-* components seem fine.

a potential fix for us later is making this test output less volatile and not binding to constant names.